snowpack/src/sources/local.ts

```diff
@@ -510,17 +510,24 @@ export class PackageSourceLocal implements PackageSource {
           .filter((spec) => spec.startsWith(packageUID))
           .map((spec) => spec.substr(packageUID.length + 1));
         // TODO: external should be a function in esinstall
-        const externalPackages = [
-          ...config.packageOptions.external,
-          ...Object.keys(packageManifest.dependencies || {}),
-          ...Object.keys(packageManifest.peerDependencies || {}),
-        ].filter((ext) => ext !== _packageName && !NEVER_PEER_PACKAGES.includes(ext));
-        const externalPackagesFull = [
-          ...externalPackages,
-          ...Object.keys(packageManifest.devDependencies || {}).filter(
-            (ext) => ext !== _packageName && !NEVER_PEER_PACKAGES.includes(ext),
-          ),
-        ];
+
+        const filteredExternal = (ext: string) => ext !== _packageName && !NEVER_PEER_PACKAGES.has(ext);
```

Comment:
`ext` here meaning `external`? I would probably rename that variable for clarity.